src/plugins/mer/merdeploysteps.cpp

```diff
@@ -153,7 +153,7 @@ class MerRpmValidationStepConfigWidget : public ProjectExplorer::BuildStepConfig
             item->setCheckState(0, step->selectedSuites().contains(suite.id) ? Qt::Checked : Qt::Unchecked);
             item->setData(0, Qt::UserRole, suite.id);
         }
-        m_ui.suitesTreeWidget->header()->setSectionResizeMode(0, QHeaderView::Stretch);
+        m_ui.suitesTreeWidget->header()->setSectionResizeMode(0, QHeaderView::ResizeToContents);
         m_ui.suitesTreeWidget->header()->setSectionResizeMode(1, QHeaderView::ResizeToContents);
         m_ui.suitesTreeWidget->header()->setSectionResizeMode(2, QHeaderView::ResizeToContents);
```

Comment:
I think we should set index 1 to Stretch now. We want the table to use all available width and at the same to avoid horizontal scrollbars to appear.

Could you please attach two screenshots - one with short URL, one with URL which overflows - after changing this?

Reply:
Screenshots with stretch on website column:
`m_ui.suitesTreeWidget->header()->setSectionResizeMode(1, QHeaderView::Stretch);`


![strech_with_long_url](https://user-images.githubusercontent.com/43814603/105368322-7494af00-5c12-11eb-836b-badc5a84a25f.png)
![strech_with_short_url](https://user-images.githubusercontent.com/43814603/105368332-79596300-5c12-11eb-8510-0be9ea518294.png)

Reply:
So, if we want to avoid horizontal scrollbars we need more changes.

Reply:
And how does it behave *without* changing resize mode for column 1?

```
         m_ui.suitesTreeWidget->header()->setSectionResizeMode(0, QHeaderView::ResizeToContents); 
         m_ui.suitesTreeWidget->header()->setSectionResizeMode(2, QHeaderView::ResizeToContents); 
```

Reply:
In this case the address overlaps the last column
![no_resize](https://user-images.githubusercontent.com/43814603/105482706-cfcdac80-5cb9-11eb-9460-b349b945ef08.png)

Reply:
pity. If you do not manage to fix it, we could work it around by swapping the second and third column, .i.e., making url the last column. But I would like to avoid that.